src/sentry/static/sentry/app/views/alerts/status.tsx

```diff
@@ -2,62 +2,49 @@ import React from 'react';
 import styled from '@emotion/styled';
 
 import {t} from 'app/locale';
-import InlineSvg from 'app/components/inlineSvg';
 import space from 'app/styles/space';
+import {IconCheckmark, IconWarning, IconFire} from 'app/icons';
 
 import {Incident, IncidentStatus} from './types';
 
 type Props = {
   className?: string;
   incident: Incident;
-  isSmall?: boolean;
 };
 
-const Status = ({className, incident, isSmall}: Props) => {
+const Status = ({className, incident}: Props) => {
   const {status} = incident;
-  const isIncidentOpen = status !== IncidentStatus.CLOSED;
   const isResolved = status === IncidentStatus.CLOSED;
   const isWarning = status === IncidentStatus.WARNING;
 
-  const icon = isResolved
-    ? 'icon-circle-check'
-    : isWarning
-    ? 'icon-warning-sm'
-    : 'icon-circle-exclamation';
+  const icon = isResolved ? (
+    <IconCheckmark color="green400" />
+  ) : isWarning ? (
+    <IconWarning color="orange400" />
+  ) : (
+    <IconFire color="red400" />
+  );
 
   const text = isResolved ? t('Resolved') : isWarning ? t('Warning') : t('Critical');
 
   return (
-    <Wrapper className={className} isSmall={!!isSmall}>
-      <Icon src={icon} status={status} isOpen={isIncidentOpen} />
+    <Wrapper className={className}>
+      <Icon>{icon}</Icon>
       {text}
     </Wrapper>
   );
 };
 
 export default Status;
 
-type WrapperProps = {status: IncidentStatus};
-
-function getColor({theme, status}) {
-  if (status === IncidentStatus.CLOSED) {
-    return theme.green500;
-  } else if (status === IncidentStatus.WARNING) {
-    return theme.yellowDark;
-  }
-
-  return theme.redDark;
-}
-
-const Wrapper = styled('div')<{isSmall: boolean}>`
-  display: flex;
+const Wrapper = styled('div')`
+  display: grid;
```

Comment:
Thanks for using grid. Can you use grid gap to get rid of the margins on the icon?